README.rst

```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
 Please provide unit tests if you contribute logic beyond bare bare business object definitions.
 
 To enhance this BO4E implementation and contribute to this project check out the `develop branch`_, install `tox`_ and set the virtual environment created by the command
-::
+.. code-block:: shell
+
    tox -e dev 
    
 The created venv should be located somewhere around .tox/dev/Scripts.
 
+Release workflow
+================
+* Check with tox all tests and lintings: `tox`
+* Check with tox if the packaging works fine: `tox -e test_packaging`
+* Merge all your changes you would like to have in the release into the master branch.
+* Check that all github actions for tests and linting do pass.
+* Go to `BO4E-python`_ and click on *Create a new release* in the right sidebar
```

Comment:
you can create a link to that form directly?

Reply:
No, I added the link at the bottom of the README.

Reply:
I thought of this link: https://github.com/Hochfrequenz/BO4E-python/releases/new